puppet/modules/jenkins_node/manifests/packaging/rpm.pp

```diff
@@ -47,8 +47,6 @@
     manage_dir => true,
   }
 
-  include rsync
```

Comment:
Don't we still need to ensure the `rsync` package is present? While we can count on the utility, I wouldn't mind it being explicit if we rely on it.

Reply:
I was under the impression https://github.com/theforeman/foreman-infra/pull/2066/files#diff-c9be9d7252cac9998dce62c020c7a0b220cf5e1c6d95c511ca9484127b3df492 would handle that.

Reply:
done